modules/virt-canceling-vm-import.adoc

```diff
@@ -0,0 +1,16 @@
+// Module included in the following assemblies:
+// * virt/virtual_machines/importing_vms/virt-importing-rhv-vm.adoc
```

Comment:
is this going to be shown in 4.6 only?

Reply:
That's what you wrote in the Jira

Reply:
I don't see the distinction from 4.5 where you have to delete the VM/VMImport to Cancel it.